config.go

```diff
@@ -87,3 +92,19 @@ func WithBetaVersion(betaVersion ...BetaVersion) ClientOption {
 		c.BetaVersion = betaVersion
 	}
 }
+
+func WithVertexAI(projectID string, location string) ClientOption {
+	return func(c *ClientConfig) {
+		c.BaseURL = fmt.Sprintf("https://%s-aiplatform.googleapis.com/v1/projects/%s/locations/%s/publishers/anthropic/models", location, projectID, location)
+		c.APIVersion = APIVersionVertex20231016
+	}
+}
+
+func WithApiKeyFunc(apiKeyFunc ApiKeyFunc) ClientOption {
+	return func(c *ClientConfig) {
+		c.apiKeyFunc = apiKeyFunc
+	}
+}
+func isVertexAI(apiVersion APIVersion) bool {
```

Comment:
I think this could be a method on the `APIVersion` type. (maybe call it `isVertex()`)

Reply:
Done